### ISSUE TITLE
Custom metric

### ISSUE THREAD

**Issue by akshitamishra1213** [shankarpandala/lazypredict#362]
Hi there, 
Thank you for this code. 
It would be nice if you could provide an example of how to use custom metrics.

**Comment by emirtarik** [comment]
I second this. All examples use `custom_metrics=None`. I tried assigning a series of strings looking at the names in the source code but no luck.

**Comment by Naycat** [comment]
I had the same problem.
The custom metrics are sklearn metrics. you can solve it like this:
`from sklearn.metrics import mean_absolute_error
reg = LazyRegressor(predictions=True, custom_metric = mean_absolute_error)
models, predictions = reg.fit(X_train, X_test, y_train, y_test)`

**Comment by BradKML** [comment]
Throwing one more to the heap but would like to re-direct you guys here https://github.com/nityansuman/lazypredict-nightly
